README.md

```diff
@@ -72,6 +72,20 @@ There's also a number of variables which are used to connect to MongoDb;
 
 When running `setup.sh`, the `.env.template` file will be copied to `.env` if the latter does not exist.
 
+## OAuth Setup
+
+This application uses OAuth for user authentication. The following GitHub documentation link shoes you how to register your application on GitHub for OAuth.
+* https://docs.github.com/en/developers/apps/creating-an-oauth-app
+
+Before running your application, the following envirment variable to be set
+* GITHUB_CLIENT_ID - Provided when you register your application on GitHub
+* GITHUB_CLIENT_SECRET - Provided when you register your application on GitHub
+* GITHUB_LOGON_REDIRECT - The URL for OAuth callback to re-direct you to. This should be the homepage URL with /login/callback appended e.g. http://localhost:5000/login/callback
+
+For development purposes it is possible to disable OAuth by setting the following enviroment variable.
+
+* OAUTHLIB_INSECURE_TRANSPORT=1
```

Comment:
I think here you mean `LOGIN_DISABLED` - this variable is for disabling restrictions that Oauth can only run over secure connections (e.g. https)